src/palette.ml

```diff
@@ -110,6 +110,31 @@ let load_tic80_palette (raw : string) : t =
   else
     raise (Invalid_argument "Palette size must not be zero or negative")
 
+(* New function for Task #27 *)
+let of_list (rgb_list : int list) : t =
+  if List.length rgb_list > 0 then
+    Array.of_list (List.map Int32.of_int rgb_list)
+  else
+    raise (Invalid_argument "Palette size must not be zero or negative")
+let load_lospec_palette (s : string) : t =
```

Comment:
Don't have functions run into each other like this - please leave one line clear between them so the code is easier to read.